src/selectors/modular-pipelines.js

```diff
@@ -0,0 +1,54 @@
+import { createSelector } from 'reselect';
+import { getPipelineModularPipelineIDs } from './pipeline';
+
+const getNodesModularPipelines = (state) => state.node.modularPipelines;
+const getModularPipelineIDs = (state) => state.modularPipeline.ids;
+const getModularPipelineName = (state) => state.modularPipeline.name;
+const getModularPipelineEnabled = (state) => state.modularPipeline.enabled;
+
+/**
+ * Retrieve the formatted list of modular pipeline filters
+ */
+export const getModularPipelineData = createSelector(
+  [getModularPipelineIDs, getModularPipelineName, getModularPipelineEnabled],
+  (modularPipelineIDs, modularPipelineName, modularPipelineEnabled) =>
+    modularPipelineIDs.sort().map((id) => ({
```

Comment:
Might be a good idea to clone the array first, since sort is in-place on the original array. Usually I do this as a pattern just to be safe when the array comes from outside: `modularPipelineIDs.slice(0).sort()`